sources/osg/CullVisitor.js

```diff
@@ -545,7 +545,12 @@ CullVisitor.prototype[ Geometry.typeID ] = ( function () {
             depth = this.distance( bb.center( tempVec ), modelview );
         }
         if ( osgMath.isNaN( depth ) ) {
-            Notify.warn( 'warning geometry has a NaN depth, ' + modelview + ' center ' + tempVec );
+
+            if ( !loggedOnce ) {
```

Comment:
What you want actually is a loggedOncePerFrame, it can be done using the frame stamp in the visitor